osu.Game/Beatmaps/IBeatmap.cs

```diff
@@ -69,6 +70,43 @@ public interface IBeatmap
         /// </summary>
         double GetMostCommonBeatLength();
 
+        double AudioLeadIn { get; internal set; }
```

Comment:
Is the `internal set;` part temporary? Maybe just `set;` is fine.

I'm not sure what the future is for this interface so I'm not suggesting a change - just asking.

Reply:
The spirit of that modifier was to have editor flows working (which depend on being able to mutate `IBeatmap`s) while keeping up a semblance of being immutable to external consumers. To what degree this matters given other things such as `Breaks` or `ControlPointInfo` are already doing away with that illusion and just doing `{ get; set; }` is disputable I guess. `set;` would be fine by me if it's fine by everyone else.